src/components/GameInfoV2/index.tsx

```diff
@@ -0,0 +1,231 @@
+}
+
+interface BlockchainIconProps {
+  icon: React.ElementType
+}
+
+export interface GameInfoV2Props {
+  title: string
+  version?: string
+  earlyAccess?: boolean
+  playerCount?: string
+  image?: {
+    src: string
+    alt: string
+    fallback?: React.ReactNode
+  }
+  blockchains?: Array<{
+    icon: React.ElementType
+    isHoverable?: boolean
+    name?: string
+  }>
+  showRemainingCount?: boolean
+  remainingCount?: string | number
+  socialLinks?: SocialLinks[]
+  editorChoice?: EditorChoiceType
+  actionButton?: React.ReactNode
+  className?: string
+  i18n?: GameInfoV2i18n
+}
+
+const GameImage: React.FC<GameImageProps> = ({
+  src,
+  alt,
+  fallback = (
+    <img
+      src={fallbackCard}
+      alt="Game cover fallback"
+      className={styles.fallbackImage}
+    />
+  ),
+  className
+}) => {
+  const [hasError, setHasError] = React.useState(false)
+
+  if (!src || hasError) {
+    return fallback ? (
+      <div className={classNames(styles.imageFallback, className)}>
+        {fallback}
+      </div>
+    ) : null
+  }
+
+  return (
+    <img
+      src={src}
+      alt={alt || 'Game cover'}
+      className={classNames(styles.image, className)}
+      onError={() => setHasError(true)}
+    />
+  )
+}
+
+const BlockchainIcon: React.FC<{
+  blockchain: BlockchainIconProps
+  className?: string
+}> = ({ blockchain, className }) => (
+  <div className={classNames(styles.blockchainIcon, className)}>
+    {React.createElement(blockchain.icon, {
+      className: styles.blockchainSvg
+    })}
+  </div>
+)
+
+const MAX_VISIBLE_BLOCKCHAINS = 5
+
+const GameInfoV2: React.FC<GameInfoV2Props> = ({
+  title,
+  version,
+  earlyAccess,
+  playerCount,
+  image,
+  blockchains = [],
+  showRemainingCount = true,
+  remainingCount = 9,
+  socialLinks,
+  editorChoice,
+  actionButton,
+  i18n,
+  className
+}): JSX.Element => {
+  const renderEditorChoice = () => {
+    if (!editorChoice?.isEditorChoice) return null
+
+    return (
+      <div className={classNames(styles.editorChoice, editorChoice.className)}>
+        <Images.EditorChoice />
+        Editor&apos;s Choice {editorChoice.year || new Date().getFullYear()}
+      </div>
+    )
+  }
+
+  const renderBlockchains = () => {
+    if (!blockchains || blockchains.length === 0) return null
+
+    const visibleBlockchains = blockchains.slice(0, MAX_VISIBLE_BLOCKCHAINS)
+
+    return (
+      <div className={styles.blockchains}>
+        <span className={styles.blockchainLabel}>
+          {i18n?.blockchainLabel || 'BLOCKCHAIN(S):'}
+        </span>
+        <div className={styles.blockchainIcons}>
+          {visibleBlockchains.map((blockchain, index) => (
+            <BlockchainIcon key={index} blockchain={blockchain} />
+          ))}
+          {showRemainingCount && (
+            <div className={styles.more}>
+              <span className={styles.moreCount}>+{remainingCount}</span>
+            </div>
+          )}
+        </div>
+      </div>
+    )
+  }
```

Comment:
let's make this a separate component

Reply:
also the on hover styling/interactions don't seem to match the design. can we check with design here? I think the desired behavior was only `Popover` on the `+X` count which would show a list of the remaining chains

Reply:
I talked with @pimentel08 and she said we have a tooltip on the count showing icon and name of the rest of the blockchains.